### PR TITLE
[Symfony UX] reference Vue component in VueJs page

### PR DESCRIPTION
**Pull request by zairigimad**

### CODE REVIEW

**Review by 94noni — APPROVED** [review]
Just question, why targeting 6.1 and not other branch?

File: frontend/encore/vuejs.rst [hunk 1]
```diff
@@ -6,6 +6,10 @@ Enabling Vue.js (``vue-loader``)
 
     Do you prefer video tutorials? Check out the `Vue screencast series`_.
 
+.. tip::
+
+    Check out live demos of Symfony UX Vue component at `https://ux.symfony.com/vue`_!
```
Comment (94noni) [inline frontend/encore/vuejs.rst:11]:
```suggestion
    Check out live demos of Symfony UX Vue.js component at `https://ux.symfony.com/vue`_!
```

Comment (94noni) [reply frontend/encore/vuejs.rst:11]: Ref https://github.com/symfony/ux-vue header :)

Comment (javiereguiluz) [reply frontend/encore/vuejs.rst:11]: Fixed while merging. Thanks.

**Comment by javiereguiluz** [comment]
Thanks @zairigimad. Merged in `4.4` branch and up.

**Pull request merged**